src/EditorFeatures/Test/Diagnostics/IDEDiagnosticIDConfigurationTests.cs

```diff
@@ -1209,6 +1212,9 @@ public void VisualBasic_VerifyIDECodeStyleOptionsAreConfigurable()
 # IDE0120
 No editorconfig based code style option
 
+# IDE0131
+No editorconfig based code style option
```

Comment:
we would want an editorconfig value for this.  otherwise people can't disable the hint.  or, alternatively, some projects might want to make it an error if you use the old form and not the new form.